test/ponder/serialise.cpp

```diff
@@ -150,21 +150,23 @@ TEST_CASE("Can serialise using RapidXML")
         std::string serialised;
         
         {
-            std::unique_ptr<Ref> r{ new Ref };
-            REQUIRE(r != nullptr);
-            
-            r->m_instance.m_i = 89;
-            r->m_instance.setF(0.75f);
-            r->m_instance.m_s = "stringy";
-
+            std::unique_ptr<Simple> s1{ new Simple };
```

Comment:
The point of this test is that we save a nested object instance. This has been changed and will just test the same as the previous test.